docs/buildah-build.1.md

```diff
@@ -910,7 +910,7 @@ in a safe way that will not end up stored in the final image, or be seen in othe
 The value of the secret will be read from an environment variable or file named
 by the "id" option, or named by the "src" option if it is specified, or from an
 environment variable specified by the "env" option.
-The secret will be mounted in the container at `/run/secrets/*id*` by default.
+The secret will be mounted in the container at `/run/secrets/id` by default.
```

Comment:
Removing the emphasis here seems wrong to me, but I don't see how to get it to render correctly (as something other than a literal `*`, which is what we get now) in the man page.

Reply:
Yeah, I removed the asterisk here because it wasn't rendering as emphasis, and I admittedly made an assumption that additional emphasis had been the intention of the `*` in the first place. I can apply one of @flouthoc's suggestions (or any) if that is preferred.

Reply:
I'd prefer keeping the `*`, but since the resolve issues can't be conquered, I'd go with `<id>` as we do in a few other places like:
https://github.com/containers/buildah/blob/2517aa202fc25bb2c4a6cfc545df8e55dda7959e/pkg/cli/common.go#L304

Reply:
I applied @TomSweeneyRedHat 's suggestion, rebased and pushed the branch. Tests passing. Let me know if there is anything else I need to do!